src/main/kotlin/no/nav/syfo/consumer/pdl/PdlPersonResponse.kt

```diff
@@ -111,3 +114,27 @@ fun PdlHentPerson.getName(): String? {
         }
     }
 }
+
+fun PdlHentPerson.bostedsadresse(): Bostedsadresse? {
+    val bostedsadresse = this.hentPerson?.bostedsadresse
+    if (bostedsadresse.isNullOrEmpty()) {
+        return null
+    }
+    return bostedsadresse.first()
+}
```

Comment:
Kan dette forenkles til `return this.hentPerson?.bostedsadresse?.first() ?: null` eller noe? Det ble ikke så fint som jeg tenkte opprinnelig da 🤷‍♂️

Reply:
Det er godt mulig det, skal sjekke det ut.

Reply:
`public fun <T> List<T>.first(): T {
    if (isEmpty())
        throw NoSuchElementException("List is empty.")
    return this[0]
}`

Reply:
Okei, kjekt å vite 👍 Det er uansett ikke krise, bare fint de gangene Kotlin kan gjøre koden litt enklere!

Reply:
Sant sant 👍